features/cellular/framework/API/CellularInformation.h

```diff
@@ -21,7 +21,8 @@
 #include <stddef.h>
 #include "nsapi_types.h"
 
-namespace mbed {
+namespace mbed
+{
```

Comment:
as it was  - it was good ( see astyle in the travis )

Style changes, own commit please

Reply:
This follows Mbed OS coding guidelines, see astyle in https://os.mbed.com/teams/SDK-Development/wiki/mbed-sdk-coding-style

Reply:
That is the old one. Please do not use it. Anyway, from there "Braces - K&R (see the exception 1 TBS below)" . this `{` should be attached.

Use https://os.mbed.com/docs/latest/reference/style.html (points to astylerc file that is in our codebase) . Travis runs this and can show you how this breaks  the style defined there